app/models/pod.rb

```diff
@@ -25,7 +25,9 @@ class Pod < ActiveRecord::Base
 
   class << self
     def find_or_create_by(opts) # Rename this method to not override an AR method
-      u = URI.parse(opts.fetch(:url))
+      url = opts.fetch(:url)
```

Comment:
Maybe then I can add a check for Rails.env.test? here? And exit silently on url.nil? if we are on test?